uaclient/gpg.py

```diff
@@ -46,6 +41,16 @@ def export_gpg_key_from_keyring(
         raise exceptions.UserFacingError(
```

Comment:
Not introduced in this branch, but the line just above checks `err` for the english text `nothing exported`, and that is locale dependent:
```
root@trusty-ua:~# apt-key export 123123
gpg: AVISO: nada exportado
```

Reply:
We've changed the code to no longer do this, so I'm going to mark this resolved.